tests/unit/test_column_noise.py

```diff
@@ -20,7 +20,7 @@
 @pytest.fixture(scope="module")
 def dummy_dataset():
     # Add a column of integer strings
-    num_simulants = 1_000_000
+    num_simulants = 100_000
```

Comment:
Did we decide 100k is our threshold?

Reply:
I think we can leave at 1 million unless the tests are too slow.